packages/components/src/VirtualizedList/CellDatetime/CellDatetime.test.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import CellDatetime from './CellDatetime.component';
+
+describe('CellDatetime', () => {
+	it('should render with "ago"', () => {
+		// when
+		const columnData = {
+			data: {
+				mode: 'ago',
+			},
+		};
+
+		const wrapper = shallow(<CellDatetime cellData={1518561116} columnData={columnData} />);
+
+		// then
+		expect(wrapper.getElement()).toMatchSnapshot();
```

Comment:
would prefer a better test checking if the produced value is correct, or to test computeValue wich is exported

guess something wrong is going on https://github.com/Talend/ui/pull/1057/files#diff-b77d66e1616a142ea611a5f02174cdf7R7

Reply:
✅